include/player.hpp

```diff
@@ -38,8 +38,9 @@ class Player : public DynamicObject, public InputManager::Input
 
     virtual void hardInit(Dynamics*);
     virtual void mouseCallback(GLFWwindow*, double xpos, double ypos);
+    virtual void mouseButtonCallback(GLFWwindow*, int button, int action, int mods);
     virtual void pollInput(GLFWwindow*, float dt);
-
+    virtual int getPoints();
```

Comment:
points is a temporary concept used to track "hits" with the hitscan sniper rifle